ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -220,7 +225,12 @@
                 &lt;/div&gt;
               &lt;/form&gt;
             #else
-              #set($url = $xwiki.getURL("XPoll.XPoll for the Meeting $doc.name"))
+              #set($meetingName = $doc.name)
+              #set($defaultDocName = $services.model.getEntityReference('DOCUMENT', 'default').name)
+              #if($meetingName == $defaultDocName)
+                #set($meetingName = $doc.documentReference.lastSpaceReference.name)
+              #end
```

Comment:
You could extract the duplicated code in a macro that returns the poll title.

Reply:
Yes, I forgot that.